data/pages-content/tickets.md

```diff
@@ -45,131 +32,107 @@ Each of the three ticket types is available in three payment tiers.
 We encourage and trust you to pick a fair ticket tier that fits your personal
 circumstance. The money you spend will be put straight back into the community
 to support our diversity initiatives,
-[financial aid programmes](/finaid) and
+[financial aid programmes](https://ep2023.europython.eu/finaid) and
 [other grants](https://www.europython-society.org/grants/)
 offered to the European Python community.
 
--   **Business Tickets**: for attendees whose company/business is paying for
-    them to attend, or if you use Python professionally. As someone with the
-    means to afford this tier you help us keep the conference affordable for
-    everyone. Thank you!
--   **Personal Tickets**: for those who enjoy Python as a hobbyist or use it as
-    a freelancer.
--   **Education Tickets**: for students, and teachers working in the education
-    sector (your main employment is at a school, college or university).
+- **Business Tickets**: for attendees whose company/business is paying for
+  them to attend, or if you use Python professionally. As someone with the
+  means to afford this tier you help us keep the conference affordable for
+  everyone. Thank you!
+- **Personal Tickets**: for those who enjoy Python as a hobbyist or use it as
+  a freelancer.
+- **Education Tickets**: for students, and teachers working in the education
+  sector (your main employment is at a school, college or university). We offer Education Tickets below costs.
 
 **Note**: If you need a proper VAT invoice listing your company's name, please
 purchase a business ticket. Personal Tickets do not include a company name. We
 will list the ticket tier and type on your conference badge to accommodate the
 registration staff.
 
-<a class="button" href="https://tickets.europython.eu">Get your tickets here</a>
 
-<h2 className="soldout"> Early Bird Sales</h2>
+<button>Tickets available soon</button>
+
 
-We're offering a flat 33% discount across all tiers for Early Bird Conference
+## Early Bird Ticket Sales
+
+We're offering a flat #FIXME discount across all tiers for Early Bird Conference
 Tickets!
 
-We have a total of 250 Early Bird tickets (130 Business, 80 Personal, 40
+We have a total of #FIXME Early Bird tickets (#FIXME Business, #FIXME Personal, #FIXME
 Education) available when we open up ticket sales on
-**Thursday, 17th March 2022 13:00 CET**. These tickets will be obtained on a
+**Tuesday, 21 March 2022 #FIXME**. These tickets will be obtained on a
 first come, first serve basis. When they're gone, they're all gone!
 
 We'll switch to regular ticket sales as soon as Early Bird tickets are sold
 out.
 
+**NOTE**: During our last in-person conference at Basel in 2019, the Early Bird
```

Comment:
last in-person conference was 2022, Dublin.

Reply:
:+1: What @Vaibhavs10 said